openbas-api/src/main/java/io/openbas/rest/asset/endpoint/form/EndpointExportImport.java

```diff
@@ -0,0 +1,51 @@
+package io.openbas.rest.asset.endpoint.form;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.opencsv.bean.CsvBindByName;
+import com.opencsv.bean.CsvBindByPosition;
+import io.openbas.database.model.Endpoint;
+import lombok.Data;
+
+@Data
+public class EndpointExportImport {
+
+  @JsonProperty("asset_name")
+  @CsvBindByName(column = "asset_name", required = true)
```

Comment:
Why is there both annotations ? How does opencsv resolve this if say asset_name is not in position 0 ?